help/user-guide/isa-json-compliant-experiment.md

```diff
@@ -0,0 +1,117 @@
+---
+ title: SEEK User Guide - ISA-JSON compliant Experiment
+ layout: page
+---
+
+# ISA-JSON compliant Experiment
```

Comment:
I suggest to put a disclaimer in the general SEEK docs that the information on this page refers to a feature that can be turned on via the admin panel.